--- a/engine/checkers_game.py
+++ b/engine/checkers_game.py
@@ -4,6 +4,7 @@
 import queue
 from .board import Board
 from .constants import SQUARE_SIZE
+import engine.constants as constants
 from gui.button import Button
 from engine.search import minimax_alpha_beta_search
 
@@ -19,6 +20,10 @@
         self.selected_piece = None
         self.valid_moves = {}
         self.game_over = False
+
+        # Correctly load assets after pygame has been initialized
+        constants.CROWN = pygame.transform.scale(pygame.image.load(constants.CROWN_PATH), (44, 25))
+
         self._update_valid_moves() # Calculate initial valid moves for white
         self.font = pygame.font.SysFont(None, 36)
